horace_core/sqw/coord_transform/@projaxes/private/check_and_set_type_.m

```diff
@@ -2,20 +2,17 @@
 % Verify if type argument set to projaxes class is correct
 % and set type variable if it does
 
-ok=true;
-mess = '';
 if is_string(type) && numel(type)==3
     type=lower(type);
-    if isempty(strfind('arp',type(1))) || isempty(strfind('arp',type(2))) || isempty(strfind('arp',type(1)))
-        mess='Normalisation type for each axis must be ''r'', ''p'' or ''a''';
-        ok=false;
+    if any(arrayfun(@(t)(~contains('arp', t)), type))
```

Comment:
You are failing to reproduce the probable bug at L.9 on the left where type(1) gets tested twice and type(3) not at all. Which is probably correct. But just checking.